### PR TITLE
Fix broken impl download link

### PR DESCRIPTION
**Pull request by patrickbkr**
Remove a broken and mostly redundant link. Some minor cleanup.
Fixes rakudo/rakudo.org#105.

### CODE REVIEW

File: source/compilers/index.html [hunk 1]
```diff
@@ -17,19 +17,17 @@ <h2><b>Raku</b> Compilers</h2>
...
-        <h4><a href="https://rakudo.org/">Rakudo™</a> and <a href="https://github.com/raku/nqp/">NQP</a></h4>
+        <h4><a href="https://rakudo.org/">Rakudo™</a> and NQP</h4>
         <p>Rakudo runs on
         <a href="http://moarvm.org/">MoarVM</a> and the Java Virtual Machine
         (JVM).
-        It is actively developed on the
-        <a href="http://github.com/rakudo/rakudo">Rakudo GitHub</a> site. <a
-        href="https://rakudo.org/files">Download or compile
-        Rakudo</a>.</p>
+        It's developed on <a href="http://github.com/rakudo/rakudo">GitHub</a>.
```
Comment (lizmat) [inline source/compilers/index.html:24]: what happened to "actively" ?

**Comment by patrickbkr** [comment]
My thought was that "actively" is a bit random here. Especially given there is a big blog of "dormant" implementations below.

**Comment by coke** [comment]
I think it's there **because** of the list of older compilers - highlights it's the only one being developed at the moment.

**Comment by patrickbkr** [comment]
Okay, okay, I'll change it back ...

**Comment by patrickbkr** [comment]
Having slept on this, I think I know why I dislike the "actively" there. It kind of implies that stuff you see on raku.org might be stale. (Otherwise why would it be necessary to say that this one bit is not dead?) I don't like that implication. Also it shouldn't be necessary to state that we're alive. That also adds the implication that we're kind of struggling. As in "It might look like it, but no, we are active."

**Comment by lizmat** [comment]
Fair enough: I just wanted to make sure it wasn't removed inadvertently

**Pull request merged**